src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -100,6 +118,55 @@ internal static string GetAccountId(string functionArn)
             return null;
         }
 
+        internal static string GetFaasTrigger<TInput>(TInput input)
+        {
+            var trigger = "other";
+            if (input is APIGatewayProxyRequest || input is APIGatewayHttpApiV2ProxyRequest)
+            {
+                trigger = "http";
+            }
+
+            return trigger;
+        }
+
+        internal static IEnumerable<KeyValuePair<string, object>> GetFunctionDefaultTags<TInput>(TInput input, ILambdaContext context)
```

Comment:
Why "Default"?

Reply:
Initially I thought about adding optional tags list to Trace methods and merging them with these tags. That's I called them "Default" (means if optional are not provided then these will be added). But optional tags are is not needed anymore. I'll remove "Default".